data/publiccloud/log_instance.sh

```diff
@@ -28,7 +28,9 @@ ec2_stop_log()
     aws ec2 get-console-output --instance-id "$INSTANCE_ID" --output text > "${OUTPUT_DIR}/${CNT}_get_console_output_stop.log"
     set -e
     local pids=$( test -f "$PID_FILE" && cat "$PID_FILE")
-    rm "$PID_FILE"
+    if [ -f "$PID_FILE" ]; then
+      rm "$PID_FILE"
+    fi
 
```

Comment:
I'd personally use oneliner: `[ -f "$PID_FILE" ] && rm "$PID_FILE"`

Reply:
I also wonder if can be useful to know when file is not present for debugging. Maybe add `-v` to rm to see that it actually removed the file? Because it's strange that why it was missing and what removed it.